misc/drone-scripts/setup.sh

```diff
@@ -33,8 +33,20 @@ do
     sleep 1;
 done
 
-echo "Reset complete"
-sleep 5;
+echo "Reset complete\nWaiting for VM to get IP..."
+
+until govc vm.ip photon.vmfs
```

Comment:
I am always scared with unbounded sleeps. Please add some timeout and fail the run.